src/pages/Home.jsx

```diff
@@ -1,25 +1,29 @@
-import Announcement from '../components/Announcement';
```

Comment:
Why so many, changes?

Reply:
Because I was using tailwind css so there was some changes in every file and again switching from that to styled component I had to uninstall tailwind CSS and it's configuration which leads so many changes.